_pages/infer-groups.html

```diff
@@ -0,0 +1,45 @@
+---
+layout: page
+longtitle: "Facebook Year Group pages"
+title: "Year Groups"
+years:
+  - 2019
+  - 2018
+  - 2017
+  - 2016
+---
```

Comment:
I think doing this:

```
links:
    - title: Incoming 2017/18
       url: ...
    (...)
    - title: Class of..
       url: ...
```

might be cleaner?

Reply:
Probably, I had a `for` loop previously just iterating from 2016-9, but it was ascending, rather than descending